core/extensions/impl/CMakeLists.txt

```diff
@@ -8,6 +8,7 @@ add_library(crypto_extension
     )
 target_link_libraries(crypto_extension
     iroha::ed25519
+    sr25519
```

Comment:
it works only because you've made "sudo make install" for sr25519. you need to make an imported cmake target, which will import sr25519 build artifacts

Reply:
I did not make install. I marked sr25519 as GLOBAL target, so it is visible.

Reply:
Ah I see